test_config.go

```diff
@@ -0,0 +1,5 @@
+package tfe
+
+type TestConfig struct {
```

Comment:
I think I would prefer if this struct were defined alongside `RegistryModuleTestConfigOptions`. WDYT?

Reply:
And perhaps this could have a similar name since TestConfig is rather generic

Reply:
Agreed. Actually, I'd like a bit of context here: 

- Do we predict other objects in TFC will have test configs of some kind? (providers, workspaces...?)
- Do we predict test configs will have additional options beyond "on / off"? (I'm guessing yes, since you went to the trouble of creating a struct, but let's lay it out explicitly while we're reviewing here!) 

Prior art: the VCSRepo struct lives over in workspace.go, even though a couple of different types nest it.

Reply:
> Agreed. Actually, I'd like a bit of context here:
> 
> * Do we predict other objects in TFC will have test configs of some kind? (providers, workspaces...?)
Yes - the backend was with that in mind.

> * Do we predict test configs will have additional options beyond "on / off"? (I'm guessing yes, since you went to the trouble of creating a struct, but let's lay it out explicitly while we're reviewing here!)
Yes - although we are not supporting any additional options at the moment. The separate struct is also reflective of there being a separate Rails model/database table on the backend.
> 
> Prior art: the VCSRepo struct lives over in workspace.go, even though a couple of different types nest it.

I'm happy to move the struct wherever you feel is best.